stub-runner/stub-runner/src/main/groovy/com/ofg/stub/mapping/StubRepository.groovy

```diff
@@ -1,16 +1,22 @@
 package com.ofg.stub.mapping
 
 import groovy.transform.CompileStatic
-import groovy.transform.PackageScope
 
 @CompileStatic
 class StubRepository {
-    @PackageScope static final String DEFAULT_DESCRIPTORS_DIRECTORY = 'mappings'
+    static final String DEFAULT_DESCRIPTORS_DIRECTORY = 'mappings'
```

Comment:
@adrian-wozniak FYI. To make it real constant in Groovy the `public` keyword (or `private`) is required (if `@PackageScope` had to be removed).